Tropos/Formatters/TRBearingFormatter.h

```diff
@@ -1,6 +1,6 @@
 @interface TRBearingFormatter : NSObject
 
-+ (NSString *)cardinalDirectionStringFromBearing:(CGFloat)bearing;
-+ (NSString *)abbreviatedCardinalDirectionStringFromBearing:(CGFloat)bearing;
++ (NSString *)cardinalDirectionStringFromBearing:(double)bearing;
++ (NSString *)abbreviatedCardinalDirectionStringFromBearing:(double)bearing;
```

Comment:
On top of that, using `CGFloat` in the model layer here adds a dependency on `QuartzCore` above just `Foundation`, which is unnecessary.